src/components/Button.js

```diff
@@ -2,18 +2,32 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = (props) => {
-  const { btnName } = props;
+  const { btnName, color, wide } = props;
+  let size = wide;
+  if (wide) {
+    size = '50%';
+  } else size = '25%';
+
   return (
-    <button type="button">{btnName}</button>
+    <button
+      type="button"
+      style={{ backgroundColor: color, width: size }}
+    >
+      {btnName}
+    </button>
   );
 };
 
 Button.propTypes = {
   btnName: PropTypes.string,
```

Comment:
Let's use `isRequired` here since `btnName` will be not be having a default prop

```suggestion
   btnName: PropTypes.string.isRequired,
```